include/mbedtls/mbedtls_config.h

```diff
@@ -3049,6 +3049,26 @@
  */
 #define MBEDTLS_PSA_ITS_FILE_C
 
+/**
+ * \def MBEDTLS_PSA_STATIC_KEY_SLOTS
```

Comment:
Please add a check in `check_config.h` to forbid having this together with dynamic key slots. As [previously discussed](https://github.com/Mbed-TLS/mbedtls/pull/9302#discussion_r1648561945), it'll probably work, but it's kind of useless and we don't want the testing burden.

Reply:
Done. But of course this adds a test dependency on https://github.com/Mbed-TLS/mbedtls/pull/9464 because now the `component_check_names` complains that `MBEDTLS_PSA_KEY_STORE_DYNAMIC` does not exists. Not a problem of course, more a note for upcoming reviewers which see the CI failing on that component.